.ci/aws/common.groovy

```diff
@@ -54,6 +54,12 @@ def wait_for_odcr_capacity(region, instance_count, odcr) {
     sh ". venv/bin/activate; ./PortaFiducia/scripts/wait_for_odcr_capacity.py --region ${region} --odcr-id ${odcr} --required-capacity ${instance_count}"
 }
 
+def p3dn_sleep_5min(instance_type) {
+    if (instance_type == "p3dn.24xlarge") {
+        sh "sleep 300"
```

Comment:
this is not a fix to the problem.  Why aren't we fixing this with a real retry?

Reply:
fixed.

Reply:
I don't see any code where you actually fixed the problem?  It looks like you just took out the hack?

Reply:
I fixed it in PortaFiducia, I will send you the CR.